CMakeLists.txt

```diff
@@ -298,15 +298,6 @@ endif()
 # E+ required libraries
 add_subdirectory(idd)
 
-execute_process(
-  COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py" "${PROJECT_SOURCE_DIR}"
-  TIMEOUT 30
-  RESULT_VARIABLE generate_epJSON_schema_result)
-if(${generate_epJSON_schema_result} MATCHES ".*timeout.*")
-  message(FATAL_ERROR "Generating epJSON Schema from IDD failed: ${generate_epJSON_schema_result}")
-endif()
```

Comment:
I feel that this should be addressed/checked in the parser itself, and not be based on elapsed time. I have had occasions where the timeout triggers prematurely. I think another PR with changes to the python script would be welcome, but it isn't really related to the dependency issues addressed here.

Reply:
In my commit, I did change it to address/check in the parser itself (but still based on elapsed time).
It could be another PR with the changes to the python script.